WMS ID 9001: Update livelabs with changes for punctuation in transcribe-live-audio

chandra-c-chud-oracle opened the pull request:
# Add here your WMS ID

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved


# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [x] All filenames are lower case (including folders, images, files, etc.)
- [x] Filenames are descriptive
- [x] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [x] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [x] Image references in markdown contain an alternate text

carmenberdant1 reviewed (commented):
Hi @chandra-c-chud-oracle, for which workshop is this PR made?

chandra-c-chud-oracle commented:
hey @carmenberdant1 , we have added a new punctuation feature for the existing live-transcribe, so I had updated the existing demos to reflect these changes, also updated some of the screenshots to reflect the current state of the console

chandra-c-chud-oracle commented:
> Hi @chandra-c-chud-oracle, for which workshop is this PR made?

I have enquired about this and added the wms id please check if this is ok, also let me know if there are any other changes required
